makefiles/Makefile.devhub-subdomain-consolidation

```diff
@@ -0,0 +1,61 @@
+GIT_BRANCH=$(shell git rev-parse --abbrev-ref HEAD)
+USER=$(shell whoami)
+
+STAGING_URL="http://docs-devhub-staging.s3-website-us-east-1.amazonaws.com"
+STAGING_BUCKET=docs-devhub-staging
+
+STRAPI_URL=http://54.219.137.111:1337
+
+PRODUCTION_BUCKET=docs-devhub-prod
+PRODUCTION_URL=https://developer.mongodb.com
+
+PATH_PREFIX=developer
+
+PROJECT=devhub
+MUT_PREFIX ?= $(PROJECT)
+REPO_DIR=$(shell pwd)
+
+COMMIT_HASH=$(shell git rev-parse --short HEAD)
+
+GITHUB_USER = $(shell printenv GITHUB_BOT_USERNAME)
+GITHUB_PASS = $(shell printenv GITHUB_BOT_PASSWORD)
+SNOOTY_DB_USR = $(shell printenv MONGO_ATLAS_USERNAME)
+SNOOTY_DB_PWD = $(shell printenv MONGO_ATLAS_PASSWORD)
+
+include ~/shared.mk
+
+update-snooty:
+	cd ../../snooty-devhub; \
+	git fetch --all; \
+	git reset --hard origin/master; \
+	npm install --production
+
+get-build-dependencies:
+	@curl https://raw.githubusercontent.com/mongodb/docs-worker-pool/meta/publishedbranches/devhub-content.yaml > ${REPO_DIR}/published-branches.yaml
+
+next-gen-html: update-snooty
+	# snooty parse and then build-front-end
+	@echo ${SNOOTY_DB_PWD} | snooty build "${REPO_DIR}" "mongodb+srv://${SNOOTY_DB_USR}:@cluster0-ylwlz.mongodb.net/snooty?retryWrites=true" --commit "${COMMIT_HASH}"; \
+	if [ $$? -eq 1 ]; then \
+		exit 1; \
+	else \
+		exit 0; \
+	fi
+	rsync -az --exclude '.git' ${REPO_DIR}/../../snooty-devhub ${REPO_DIR}; 
+	cp ${REPO_DIR}/.env.production ${REPO_DIR}/snooty-devhub; 
+	cd snooty-devhub; \
+	echo "GATSBY_SITE=${PROJECT}" >> .env.production; \
+	echo "COMMIT_HASH=${COMMIT_HASH}" >> .env.production; \
+	echo "STRAPI_URL=${STRAPI_URL}" >> .env.production; \
+	echo "PATH_PREFIX=${PATH_PREFIX}" >> .env.production; \
+	npm run build; \
+	cp -r "${REPO_DIR}/snooty-devhub/public" ${REPO_DIR};
+  
+next-gen-stage: ## Host online for review
+	mut-publish public ${STAGING_BUCKET} --prefix="${COMMIT_HASH}/${MUT_PREFIX}" --stage ${ARGS};
+	echo "${COMMIT_HASH}/${MUT_PREFIX}"
+	echo "Hosted at ${STAGING_URL}/${COMMIT_HASH}/${MUT_PREFIX}/${USER}/${GIT_BRANCH}/";
+
+next-gen-deploy:	
+	yes | mut-publish public developer-hub-prod --prefix="${MUT_PREFIX}" --deploy --deployed-url-prefix=https://mongodb.com/developer --json --all-subdirectories ${ARGS};
```

Comment:
Here is the name of the new bucket (hardcoded for now). It lives in the DevRel AWS org.